SignalShareExtension/SignalShareExtension-Bridging-Header.h

```diff
@@ -6,19 +6,19 @@
 #import <UIKit/UIKit.h>
 
 // Separate iOS Frameworks from other imports.
-#import "DebugLogger.h"
-#import "Environment.h"
-#import "OWSContactsManager.h"
-#import "OWSContactsSyncing.h"
-#import "OWSLogger.h"
-#import "OWSMath.h"
-#import "OWSPreferences.h"
-#import "Release.h"
 #import "ShareAppExtensionContext.h"
-#import "UIColor+OWS.h"
-#import "UIFont+OWS.h"
-#import "UIView+OWS.h"
-#import "VersionMigrations.h"
+#import <SignalMessaging/DebugLogger.h>
```

Comment:
This cleanup is now irrelevant, but I left it in as I prefer using the full module path, even when not required.

Reply:
+1.  IMO we should write a script that auto-fixes these, but I just can't justify the time to do it.  But if we find ourselves fixing them by hand one-by-one, that's worse.